src/axom/core/Array.hpp

```diff
@@ -297,7 +298,7 @@ class Array : public ArrayBase<T, DIM, Array<T, DIM, SPACE, StoragePolicy>>, pro
   /*!
    * \brief Constructor for transferring between memory spaces
    *
-   * \param [in] other The array in a different memory space to copy from
+   * \param [in] si The array in a different memory space to copy from
```

Comment:
```suggestion
   * \param [in] other The array in a different memory space to copy from
```

Not sure why this changed...